config.yml

```diff
@@ -8,4 +8,5 @@ blacklists:
 channels:
     - conda-forge
     - bioconda
+    - compbiocore
```

Comment:
You need to remove this - one of the main purposes of conda-forge and bioconda is to ensure packages, including all dependencies, are built in a consistent and reproducible way. To introduce other channels here defeats the point.

Reply:
In other words, if any dependencies are missing from bioconda or CF, they need to be added first, unfortunately.